frontend/src/services/fetchData.ts

```diff
@@ -66,3 +68,47 @@ export const fetchApplicationRoles = async (
     ).data;
     return applicationRoles;
 };
+
+/// Admin
+export const fetchApplicationAdmin = async (
+    applicationAdminId: number | undefined
+): Promise<FamAppAdminGet[]> => {
+    if (!applicationAdminId) return [];
+    const applicationAdmin = (
+        await AdminMgmtApiService.applicationAdminApi.getApplicationAdminByApplicationId (
+            applicationAdminId
```

Comment:
Hi @J0taFerreira , several things here:
The context you are creating the function for is important: the **name**, the input **arguments** and the **return**.

* I believe the function you created is to return more than 1. So, the function name sometimes might be better reflecting this fact (by appending 's' or other good plural name choices; otherwise developer would think (by looking at the name) the function just return 1.

* The argument you gave for your function seems indicating a wrong thing for internal function `getApplicationAdminByApplicationId`. Why would you pass `applicationAdminId` to internal function's `applicationId`?

Reply:
Hello! I followed the the famapplication-admin-api file. 

![Screenshot from 2024-01-12 14-38-51](https://github.com/bcgov/nr-forests-access-management/assets/126905995/cc37ff52-1a9c-4d13-82fe-3a47c1247347)

Reply:
My mistake, I'll fix it.

Reply:
Resolved this, code is changed for new endpoint used.